### PR TITLE
Adds support for Financial Actions API

### PR DESCRIPTION
**Pull request by martinseco**

### CODE REVIEW

File: src/api/financial/financial.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            let url = `${this.config.host}/financial-actions`;
+
+            if (parameters) {
+                const queryString = Object.keys(parameters)
+                    .map((key) => `${key}=${parameters[key]}`)
+                    .join('&');
+                url += `?${queryString}`;
+            }
+
+            const response = await http(fetch, this.config, {
```
Comment (armandojaleo) [inline src/api/financial/financial.js:34]: Can we use `getHandler` from `payments.js`

Comment (martinseco) [reply src/api/financial/financial.js:34]: I don't know if I like the idea of using something from `payments.js` here in the financial client. But maybe we can discuss if `getHandler` should be like a utils function or maybe even refactor the `http.js` to be similar to what we have in the other SDKs. WDYT?

Comment (armandojaleo) [reply src/api/financial/financial.js:34]: We could extract this common function.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "checkout-sdk-node",
-    "version": "2.1.1",
+    "version": "2.1.2",
```
Comment (Armando-Ibarra) [inline package.json:3]: Do not bump for now

**Review by Armando-Ibarra — APPROVED** [review]

File: src/api/financial/financial.js (hunk 1 above)
Comment (martinseco) [reply src/api/financial/financial.js:34]: This change is going to be implemented in a future sprint. The process will be tracked with this [task](https://checkout.atlassian.net/browse/INT-1152)

**Pull request merged**